wallet-rust/native/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "wallet-rust"
+version = "0.1.0"
+build = "build.rs"
+edition = "2018"
+
+[lib]
+name = "wallet_rust"
+crate-type = ["cdylib"]
+
+[build-dependencies]
+neon-build = "0.3.1"
+
+[dependencies]
+neon = "0.3.1"
+chain-tx-filter = { path = "../../../../chain/chain-tx-filter" }
```

Comment:
I guess these relative paths won't work on everyone's machine -- perhaps specify it against ` git = "https://github.com/crypto-com/chain.git", rev = "<git commit hash / whatever revision is needed for this>"

Reply:
ok